x-pack/plugins/observability_solution/apm/server/routes/apm_routes/register_apm_server_routes.ts

```diff
@@ -271,5 +275,13 @@ export interface APMRouteHandlerResources {
   ruleDataClient: IRuleDataClient;
   telemetryUsageCounter?: TelemetryUsageCounter;
   kibanaVersion: string;
+}
+
+export interface APMRouteHandlerResources extends RouteHandlerResources {
   getApmIndices: () => Promise<APMIndices>;
 }
+
+
+export interface APMOnlyRouteHandlerResources extends RouteHandlerResources{
```

Comment:
We can come up with better names here - the point is: one is APM index patterns only, the other one is APM+OTel.